nautobot_ssot/integrations/dna_center/diffsync/models/nautobot.py

```diff
@@ -214,10 +221,20 @@ def create(cls, adapter, ids, attrs):
         if attrs.get("tenant"):
             new_device.tenant_id = adapter.tenant_map[attrs["tenant"]]
         if attrs.get("version"):
-            new_device.custom_field_data.update({"os_version": attrs["version"]})
```

Comment:
Do I understand correctly that this effectively removes compatibility with having the software version come from the plugin rather than Nautobot? Not a problem, just something I'd like to call out a bit more explicitly in 588.removed

Reply:
This was to remove the `OS Version` CustomField as it isn't needed any longer. I thought I was clear in the changelog snippet about it being removed but happy to repharse if you can think of a clearer way of stating it.